trillian/update-trillian.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Script that updates the Trillian binaries in preparation for docker-compose
+
+set -e
+
+COMPOSE_DIR=$(pwd)
+
+# Fetch Trillian and its dependencies
+go get -d -v -u github.com/google/trillian
+cd $GOPATH/src/github.com/google/trillian
```

Comment:
Can't necessarily rely on $GOPATH, as in Go 1.8 it can be unset in order to refer to ~/go